api/envoy/extensions/rbac/audit_loggers/stream/v3/stream.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.extensions.rbac.audit_loggers.stream.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.rbac.audit_loggers.stream.v3";
+option java_outer_classname = "StreamProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/rbac/audit_loggers/stream/v3;streamv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Standard Streams RBAC Audit Loggers]
+// [#not-implemented-hide:]
+
+// Custom configuration for the RBAC audit logger that writes log entries
+// directly to the operating system's standard output.
+// The logger outputs in JSON format and is currently not configurable.
+message StdoutAuditLog {
```

Comment:
I think the directory/file names and the proto name should match.  If we're calling this `StdoutLogger`, then let's rename the directory from stream to stdout and the filename from stream.proto to stdout.proto.

Reply:
Oh, wait, just realized that these names are what is used for the stdout access logger you mentioned above.  In that case, this is fine as-is.